css-highlight-api-1/Overview.bs

```diff
@@ -482,6 +482,9 @@ Range Updating and Invalidation</h3>
 	in response to DOM changes,
 	nor can they be modified by the author after creation.
 
+	Note: In other words, the user agent is expected to store the actual
```

Comment:
@frivoal @sanketj I just noticed we never landed this.

What can we do to resolve the disagreement here? I'm inclined to agree that it's useful to have this sentence in normative text (not a note). This would specifically forbid the possibility of a UA using live range internally, while not adjusting StaticRanges passed as arguments. Without the new normative sentence, that would arguably be allowed.

Reply:
@frivoal Going to go ahead and land this. If there are any objections please let me know and I'll follow up with another PR.